sdk/react/src/lib/fragment-components/a-time-picker.tsx

```diff
@@ -0,0 +1,360 @@
+				disabled: checked
+		})} />
+		<Checkbox
+			labelText='Hide label'
+			id='hide-label'
+			checked={selectedComponent.hideLabel}
+			onChange={(checked: boolean) => setComponent({
+				...selectedComponent,
+				hideLabel: checked
+		})} />
+		<Checkbox
+			labelText='Invalid'
+			id='invalid'
+			checked={selectedComponent.invalid}
+			onChange={(checked: boolean) => setComponent({
+				...selectedComponent,
+				invalid: checked
+		})} />
+		<Dropdown
+			id="size"
+			label='Size'
+			titleText='Size'
+			items={size}
+			selectedItem={size.find(item => item.id === selectedComponent.size)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				size: event.selectedItem.id
+		})} />
+		<TextInput
+			value={selectedComponent.label}
+			labelText='Label'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				label: event.currentTarget.value
+		})} />
+		<TextInput
+			value={selectedComponent.invalidText}
+			labelText='Invalid text'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				invalidText: event.currentTarget.value
+		})} />
+		<TextInput
+			value={selectedComponent.placeholder}
+			labelText='Placeholder'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				placeholder: event.currentTarget.value
+		})} />
+		<DraggableTileList
+			dataList={[...selectedComponent.items]}
+			setDataList={updateStepList}
+			updateItem={updateListItems}
+			defaultObject={{
+				text: 'New timezone',
+				value: 'New timezoneValue'
+			}}
+			template={template} />
+	</>;
+};
+
+export const ATimePickerCodeUI = ({ selectedComponent, setComponent }: any) => <TextInput
+	value={selectedComponent.codeContext?.name}
+	labelText='Input name'
+	onChange={(event: any) => setComponent({
+		...selectedComponent,
+		codeContext: {
+			...selectedComponent.codeContext,
+			name: event.currentTarget.value
+		}
+	})}
+/>;
+
+export const ATimePicker = ({
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		rejectDrop={true}
+		{...rest}>
+			{
+				<TimePicker
+				disabled={componentObj.disabled}
+				invalid={componentObj.invalid}
+				invalidText={componentObj.invalidText}
+				placeholder={componentObj.placeholder}
+				hideLabel={componentObj.hideLabel}
+				labelText={componentObj.label}
+				light={componentObj.light}
+				size={componentObj.size}>
+					<TimePickerSelect disabled={componentObj.disabled} labelText="time-picker-1">
+						<SelectItem value="AM" text="AM" />
+						<SelectItem value="PM" text="PM" />
+					</TimePickerSelect>
+					<TimePickerSelect disabled={componentObj.disabled} labelText="time-picker-2">
+						{
+							componentObj.items.map((step: any, index: number) => <SelectItem
+								value={step.value}
+								text={step.text}
+								key={index}
+							/>)
+						}
+					</TimePickerSelect>
+				</TimePicker>
```

Comment:
User should not be able to interact (Text input/select) with the component in the fragment pane.

Recommend passing in class that removes pointer event for input & select.

Reply:
Also missing support for custom styling.